testing/bdd/features/handover/handover.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Handover algorithm
+# when a document is received with a higher value ebuttp:authorsGroupControlToken than that most recently received in the
+# currently selected sequence the Handover Manager shall switch to that document's sequence, i.e. it shall emit 
+# a document in its output sequence corresponding to the received document with the new control token without delay.
+
+    Examples:
+    | xml_file     | sequence_identifier | authors_group_identifier |
+    | handover.xml | handoverOutputSeq   | handoverTest01           |
+
+
+  # SPEC-CONFORMANCE: R24, R122
+  # Within a single sequence, all documents that contain  ebuttp:authorsGroupIdentifier shall have the same ebuttp:authorsGroupIdentifier
+  # The handover node should throw an error when this happens
+  Scenario: Different group identifiers in a sequence 
+    Given a handover node with <authors_group_identifier> and <sequence_identifier>
+    And an xml file <xml_file>
+    When it has <sequence_identifier1> and <sequence_number1>
+    And it has <authors_group_identifier1>
```

Comment:
In the table below `authors_group_identifier1` differs from the `authors_group_identifier` defined above so surely in the following steps the handover node does not produce any output based on this input? I think the error, if there is one (as opposed to ignoring the sequence altogether) is generated before line 22 "`And new document is created`".